data/polynomial.lean

```diff
@@ -94,9 +94,11 @@ lemma C_apply : (C a : ℕ → α) n = ite (0 = n) a 0 := rfl
 
 @[simp] lemma C_1 : C (1 : α) = 1 := rfl
 
-@[simp] lemma C_mul_C : C a * C b = C (a * b) :=
+@[simp] lemma C_mul_C : C (a * b) = C a * C b :=
```

Comment:
Shouldn't this be called `C_mul`, as in `mv_polynomial.lean`? Also, the next lemma is called `C_add`.